modules/kubeflow-profiles/main.tf

```
@@ -0,0 +1,48 @@
+resource local_file profiles {
+  for_each = {for profile in var.profiles : profile.namespace => profile.email}
+  content = <<EOT
+apiVersion: kubeflow.org/v1
+kind: Profile
+metadata:
+  name: ${each.key}
+spec:
+  owner:
+    kind: User
+    name: ${each.value}
+EOT
+  filename = "${path.root}/${var.argocd.path}/profiles/profile-${each.key}.yaml"
+}
+
+
+resource local_file profile_application {
+  content = yamlencode({
+    "apiVersion" = "argoproj.io/v1alpha1"
+    "kind"       = "Application"
+    "metadata" = {
+      "name"      = "profile"
+      "namespace" = var.argocd.namespace
+    }
+    "spec" = {
+      "destination" = {
+        "namespace" = "profile"
```

as I remember KF profiles not namespace-scoped,  so we can use the default namespace here
```suggestion
        "namespace" = "default"
```